### ISSUE TITLE
Custom Memory Feature Doc Issue

### ISSUE THREAD

**Issue by dakanji** [acidanthera/bugtracker#1225]
Image below from Docs appears to suggest the `Memory` items are active when `Custom Memory` is set to false 
![Screen Shot 2020-10-18 at 06 33 07](https://user-images.githubusercontent.com/1128876/96358091-12587980-110c-11eb-9709-1d18be949c51.jpg)

This seems a bit unintuitive especially since the failsafe on  Custom Memory is `false` 
Should this actually be saying `Optional when Custom Memory is true` ... similar to the other items in this section?

**Comment by vit9696** [comment]
Why? This line says that you can omit `Memory` section from your configuration when `CustomMemory` is set to `false`. This makes good sense, since when you are not requesting for custom memory configuration, you can drop the section.

**Comment by dakanji** [comment]
It is actually saying the opposite ... That the section is **active** when `CustomMemory` is set to false.
The `false` in that line should be `true` to mean what you just said ... just like the other items in that group.

**Comment by vit9696** [comment]
I disagree.

Firstly, `optional` means "not required", e.g. see [here](https://www.wordnik.com/words/optional). The part to the right is the conditional when it is not required. In plain English you can read it as "Memory entry is optional (not required) when CustomMemory is set to false".

Secondly, other items in the group have different optional condition. I.e. SMBIOS is optional (not required) when Automatic is set to true.

**Comment by dakanji** [comment]
> I disagree.

Understood. I also disagree but made the changes requested to the PR.

Pretty sure quite a few others will fall into setting it to "true" thinking this switches it off as I did. 
Not a big deal to reboot after the crash and set to "false"!
